h/static/scripts/directive/markdown.coffee

```diff
@@ -349,6 +367,7 @@ module.exports = ['$filter', '$sanitize', '$sce', '$timeout', ($filter, $sanitiz
       scope.preview = false
       output.style.height = ""
       ctrl.$render()
+      $timeout -> scope.checkLength()
```

Comment:
Wherever you are using `$timeout` to allow the entire DOM subtree to be evaluated by Angular you should investigate whether `$scope.$evalAsync` works better. Otherwise, `$timeout` will introduce unnecessary delays digest cycles. `evalAsync` will batch, whereas every `$timeout` is individually scheduled by the JS runtime through a call to `window.setTimeout`.